gemfire-shared/src/main/java/com/gemstone/gemfire/internal/shared/LauncherBase.java

```diff
@@ -300,21 +300,22 @@ protected void setDefaultVMArgs(Map<String, Object> map, boolean hostData,
     if (maxHeapStr != null && maxHeapStr.equals(this.initialHeapSize)) {
       String criticalHeapStr = (String)map.get(CRITICAL_HEAP_PERCENTAGE);
       if (criticalHeapStr == null) {
-        // for larger heaps, keep critical as 95% and 90% for smaller ones;
+        // for larger heaps, keep critical as 95-99% and 90% for smaller ones;
         // also limit memory remaining beyond critical to 4GB
```

Comment:
Is this comment now nor more applicable ("limit memory remaining beyond critical to 4GB")?